rubygem-bastion/rubygem-bastion.spec

```diff
@@ -80,18 +76,8 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-mkdir -p %{buildroot}%{foreman_bundlerd_dir}
-
-cat <<GEMFILE > %{buildroot}%{foreman_bundlerd_dir}/bastion.rb
-gem 'bastion'
-gem 'less-rails'
-GEMFILE
-
-%foreman_precompile_plugin -s
 %foreman_bundlerd_file
-
-mkdir -p %{buildroot}%{foreman_dir}/public/assets
-ln -s %{foreman_assets_plugin} %{buildroot}%{foreman_dir}/public/assets/bastion
```

Comment:
This symlink is still in the %files list, which causes the build to fail.  Is it still required, or can I safely remove it?  I can't see what purpose it might have served.